src/core/Akka.Remote/EndpointManager.cs

```diff
@@ -492,196 +543,248 @@ protected void Accepting(object message)
                         .ContinueWith(x =>
                         {
                             return new ManagementCommandAck(x.Result.All(y => y));
-                        },
-                            TaskContinuationOptions.ExecuteSynchronously)
+                        }, CancellationToken.None, TaskContinuationOptions.ExecuteSynchronously, TaskScheduler.Default)
                         .PipeTo(sender);
-                })
-                .With<Quarantine>(quarantine =>
-                {
+                });
+
+            Receive<Quarantine>(quarantine =>
```

Comment:
This method became much larger since we last looked at the JVM Remoting implementation - the role of Quarantining connections has changed quite a bit.